### PR TITLE
feat: add vyper.fun

### PR DESCRIPTION
**Pull request by vasa-develop**
License: MIT
Signed-off-by: Vaibhav Saini <vasa@dappkit.io>

## Description
This PR aims to add Vyper.fun (a game-based educational website for Vyper) to the "Interactive game tutorials" section of [learning-tools](https://ethereum.org/en/developers/learning-tools) page.

## Current Issue Faced

The website is breaking when run using `npm start` as the [`data`](https://github.com/vasa-develop/ethereum-org-website/blob/dev/src/pages-conditional/developers/learning-tools.js#L72) object receives an empty `vyperfun` object (which expects a [`childImageSharp`](https://github.com/vasa-develop/ethereum-org-website/blob/dev/src/pages-conditional/developers/learning-tools.js#L123) child object).

### CODE REVIEW

File: src/pages-conditional/developers/learning-tools.js [hunk 1]
```diff
@@ -116,6 +116,14 @@ const LearningToolsPage = ({ data }) => {
       alt: "Open Zeppelin Ethernaut",
       background: "#4F62DC",
     },
+    {
+      name: "Vyper.fun",
+      description: "page-build-vyperfun-description",
+      url: "https://vyper.fun",
+      image: data.vyper.childImageSharp.fixed,
```
Comment (ryancreatescopy) [inline src/pages-conditional/developers/learning-tools.js:123]: I believe this should be `image: data.vyperfun.childImageSharp.fixed,`

File: src/intl/en.json [hunk 2]
```diff
@@ -53,6 +53,7 @@
   "page-build-crypto-pizza-link-text": "See crypto pizza",
   "page-build-crypto-pizza-title": "Crypto Pizza",
   "page-build-cryptozombies-description": "Learn Solidity building your own Zombie game.",
+  "page-build-vyperfun-description": "Learn Vyper building your own Pokemon game.",
```
Comment (ryancreatescopy) [inline src/intl/en.json:56]: I believe this should be `Pokémon`

**Review by ryancreatescopy — CHANGES_REQUESTED** [review]

**Comment by vasa-develop** [comment]
@ryancreatescopy Pushed the changes requested. But still, somehow the UI is breaking.

**Comment by vasa-develop** [comment]
@ryancreatescopy Sorry for bugging you, but feel free to give it a look and merge this if it' all ok :smile:

**Review by samajammin — APPROVED** [review]
Thanks for this contribution @vasa-develop!

**Comment by samajammin** [comment]
@all-contributors please add @vasa-develop for content.

**Pull request merged**